tailwind.config.js

```diff
@@ -3,6 +3,19 @@ module.exports = {
     "./pages/**/*.{js,ts,jsx,tsx}",
     "./components/**/*.{js,ts,jsx,tsx}",
   ],
-  theme: { extend: {} },
+  theme: {
+    extend: {
+      colors: {
+        chili: "#C23100",
+        "international-orange": "#FF5A00",
+        koromiko: "#FFb95E",
+        "dark-metal": "#202328",
+        "dark-granite": "#65615E",
+        "light-granite": "#D4D1C5",
+        parsley: "#1D531D",
+        "apple-green": "#3DA639",
```

Comment:
Just a suggestion, can't we name this like, primary, secondary, tertiary etc... for ease understanding 😅

Reply:
Would also prefer this